### PR TITLE
Creating templates guidance + personalisation updates + text message pricing update

### PR DESCRIPTION
**Pull request by emmagifford**
## Description

Added /using-nhs-notify/nhs-app-templates, /using-nhs-notify/email-templates, /using-nhs-notify/text-message-templates and /using-nhs-notify/letter-templates. 

These are based on the Confluence pages https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?spaceKey=RIS&title=3.8+Creating+letter+templates onwards, with some minor tweaks to word ordering. Romy has approved this content now.

Uploaded the standardised Word doc templates for all 4 channels to /assets/worddocs, and linked to these. Craig in cyber security approved this approach (see ticket for screenshot).

Updated /using-nhs-notify/personalisation in line with the Confluence page. The FAQs from onboarding indicated there's a user need to provide the PDS personalisation fields. 

I also added content on how users provide their own personalisation data, in line with Tan's Confluence page https://nhsd-confluence.digital.nhs.uk/display/RIS/NHS+Notify+%7C+Personalisation+Fields.

In /pricing/text-messages, I replaced 'it will count as more than one message' with 'it will be charged as more than one message'. This was prompted by a comment from Romy on the text message template content. I tweaked the content to match the later version of the UI prototype, which we based on findings from user research.

## Context

See https://nhsd-jira.digital.nhs.uk/browse/CCM-6274 and https://nhsd-jira.digital.nhs.uk/browse/CCM-5823 for the context. The templating guidance is an interim solution to improve the template submission process for users while the UI is being developed.

## Type of changes

- [x] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: docs/pages/using-nhs-notify/letter-templates.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+- [other language letter (right aligned)](/assets/worddocs/Letter%20template%20NHS%20Notify%20-%20other%20language%20right%20aligned.docx)
+- [parent or guardian letter template](/assets/worddocs/Letter%20template%20NHS%20Notify%20-%20parent%20or%20guardian.docx)
+
+You can save the template and edit it with your message content.
+
+## Formatting your letter
+
+Do not edit the page margins in the letter template. It has the correct formatting and layout for printing.
+
+Organise your message content using the styles in the template.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/letter-templates.md:33]: Replace organise with 'format'

File: docs/pages/using-nhs-notify/letter-templates.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+## Personalisation
+
+To add a personalisation field, include a placeholder with double brackets around it. For example:
+
+{% include components/inset-text.html
+    text='Hello ((firstName)), your NHS Number is ((nhsNumber))'
+%}
+
+Read about the [personalisation fields]({% link pages/using-nhs-notify/personalisation.md %}) that are available in the Personal Demographics Service (PDS) and how to provide your own personalisation data.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/letter-templates.md:66]: Can we shorten this to entire sentence to 'Read more about using personalisation' - we can explain the relationship with PDS there instead.

File: docs/pages/using-nhs-notify/letter-templates.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+## Font and font size
+
+The font size for standard letters is 12 points. The font size for large print is 16 points.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/letter-templates.md:70]: Wonder if we should switch these sentences around as the title is 'Font and font size' but we begin talking about font size.

File: docs/pages/using-nhs-notify/letter-templates.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+NHS Notify prints letters in the font Noto Sans.
+
+### Downloading your font
+
+Noto Sans is a free font.
+
+Download Noto Sans to ensure that you can edit and view your template as it will be printed.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/letter-templates.md:78]: Could we put this in a details component? Technically downloading Noto Sans is an optional step to ensure that the template users create is as accurate to the real thing as possible.

File: docs/pages/using-nhs-notify/nhs-app-templates.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+## Personalisation
+
+To add a personalisation field, include a placeholder with double brackets around it. For example:
+
+{% include components/inset-text.html
+    text='Hello ((firstName)), your NHS Number is ((nhsNumber))'
+%}
+
+Read about the [personalisation fields]({% link pages/using-nhs-notify/personalisation.md %}) that are available in the Personal Demographics Service (PDS) and how to provide your own personalisation data.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/nhs-app-templates.md:50]: Can we shorten this to entire sentence to 'Read more about using personalisation' - we can explain the relationship with PDS there instead.

File: docs/pages/using-nhs-notify/personalisation.md [hunk 6]
```diff
@@ -18,4 +18,66 @@ To personalise a message, use double brackets to add a placeholder to your conte
...
+
+You can provide your own personalisation data.
+
+Do not send us personalisation data that's already available in PDS. We'll use the PDS data over your own data in this case.
+
+### If you're using the API
+
+Include a personalisation block in your API request.
+
+For example, if you wanted to include 'practice' as a personalisation field, the personalisation block would be:
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/personalisation.md:58]: Maybe 'GP practice' or 'GP surgery' as 'practice' on its own felt out of place on first time read.

File: docs/pages/using-nhs-notify/text-message-templates.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+## Personalisation
+
+To add a personalisation field, include a placeholder with double brackets around it. For example:
+
+{% include components/inset-text.html
+    text='Hello ((firstName)), your NHS Number is ((nhsNumber))'
+%}
+
+Read about the [personalisation fields]({% link pages/using-nhs-notify/personalisation.md %}) that are available in the Personal Demographics Service (PDS) and how to provide your own personalisation data.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/text-message-templates.md:52]: Can we shorten this to entire sentence to 'Read more about using personalisation' - we can explain the relationship with PDS there instead.

File: docs/pages/using-nhs-notify/text-message-templates.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+nav_order: 3
+permalink: /using-nhs-notify/text-message-templates
+section: Writing a message
+---
+
+Upload a Word document of your text message and NHS Notify will send it for you.
+
+## How to edit your text message using our template
+
+Download our [text message template](/assets/worddocs/Text%20message%20template%20NHS%20Notify%202.docx) to set up your email.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/text-message-templates.md:17]: 'email' at the end of sentence, replace with 'text message' - copy and paste error

File: docs/pages/using-nhs-notify/text-message-templates.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+You can save the template and edit it with your message content.
+
+## Formatting your text message
+
+Organise your message content using the styles in the template.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/text-message-templates.md:23]: Replace 'organise' with 'format'.

File: docs/pages/using-nhs-notify/text-message-templates.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+If you need to send an attachment, host the attachment separately on a website and add a link to it in your message content.
+
+## Images
+
+We do not currently support images in text messages.
+
+If you need to send an image, host it separately on a website and add a link to it in your message content.
+
+## Telling recipients who your email is from
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/text-message-templates.md:98]: Replace with 'Tell recipients who your **text message** is from' - copy and paste error

File: docs/pages/using-nhs-notify/text-message-templates.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+Find out how to set up your [reply-to email address]({% link pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md %}). This is the email address that your emails are sent from and that recipients can reply to.
```
Comment (jakecosgrove) [inline docs/pages/using-nhs-notify/text-message-templates.md:100]: Replace all mentions of email with text message -copy and paste error

You might also want to replace the last sentence with content specific to how text message sender names appear.

**Review by jakecosgrove — CHANGES_REQUESTED** [review]
This is a really strong piece of content, well done!

I've left a couple of inline comments in places, some are suggestions that outright changes, so happy to for whatever you decide there.

There's a couple of easy-fix copy and paste errors on /text-message-templates - I've left inline comments where those are.

File: docs/pages/using-nhs-notify/email-templates.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+
+## Personalisation
+
+To add a personalisation field, include a placeholder with double brackets around it. For example:
+
+{% include components/inset-text.html
+    text='Hello ((firstName)), your NHS Number is ((nhsNumber))'
+%}
+
+Read about the [personalisation fields]({% link pages/using-nhs-notify/personalisation.md %}) that are available in the Personal Demographics Service (PDS) and how to provide your own personalisation data.
```
[the remark this answers is not in the document]
Comment (emmagifford) [reply docs/pages/using-nhs-notify/email-templates.md:51]: Updated for all 4 channel pages.

File: docs/pages/using-nhs-notify/letter-templates.md [hunk 13]
```diff
@@ -0,0 +1,191 @@
...
+section: Writing a message
+---
+
+Upload a Word document of a letter and NHS Notify will print and post it for you.
+
+## How to edit your letter using our template
+
+Your Word document must meet our letter specification. The page size and layout is A4 portrait (210 x 297mm).
+
+Download the relevant letter template to set up your letter:
```
[the remark this answers is not in the document]
Comment (emmagifford) [reply docs/pages/using-nhs-notify/letter-templates.md:19]: Updated.

File: docs/pages/using-nhs-notify/letter-templates.md (hunk 1 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/letter-templates.md:33]: Done.

File: docs/pages/using-nhs-notify/letter-templates.md (hunk 2 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/letter-templates.md:66]: Done.

File: docs/pages/using-nhs-notify/letter-templates.md (hunk 3 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/letter-templates.md:70]: Or I could just break the sections into Font and Font size. I'll do that!

File: docs/pages/using-nhs-notify/nhs-app-templates.md (hunk 5 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/nhs-app-templates.md:50]: Done.

File: docs/pages/using-nhs-notify/text-message-templates.md (hunk 7 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/text-message-templates.md:52]: Done.

File: docs/pages/using-nhs-notify/text-message-templates.md (hunk 8 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/text-message-templates.md:17]: Sorted.

File: docs/pages/using-nhs-notify/letter-templates.md (hunk 4 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/letter-templates.md:78]: Good idea - I've done that now. (Thanks for the help with figuring out the invalid syntax!)

File: docs/pages/using-nhs-notify/text-message-templates.md (hunk 10 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/text-message-templates.md:98]: Done.

File: docs/pages/using-nhs-notify/text-message-templates.md (hunk 11 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/text-message-templates.md:100]: Changed all mentions to 'text message'. I've not added the sentence about how text message sender names appear as this content isn't currently detailed on the sender names page.

File: docs/pages/using-nhs-notify/personalisation.md (hunk 6 above)
Comment (emmagifford) [reply docs/pages/using-nhs-notify/personalisation.md:58]: This is a good point. It triggered a whole new conversation about whether spaces are acceptable in personalisation fields. Chris thinks they're fine but he's going to run tests tomorrow to double check.

In the meantime, I've changed the personalisation values in the example components to just be 'GP'. Hope this is okay in the interests of time. I can create a new PR when we know for sure.

**Review by emmagifford — COMMENTED** [review]
As well as implementing all the changes Jake suggested, I made the following additional edits. 

1. Changed content about starting links with https:// to advise users to start with www instead, on /using-nhs-notify/letter-templates. 
 
This followed a conversation with Clare, who advised that our current letter suppliers' 'templates are based on our internal html/css pdf generation, so we can present links however we like, so guess the simplest solution is to switch the guidance to drop the https://.

2. Removed paragraph from NHS App page about using the full URL if the recipient is not expecting to receive message - waiting to hear from Harry in NHS App team about this so it's safer to remove it for now.

3. On /using-nhs-notify/personalisation I changed the personalisation values in the example components to just be 'GP' for the time being. Previously it was 'practice', but I can't change it to 'GP surgery' or 'GP practice' without knowing the stance from the devs.

Jake's comment triggered a whole new conversation about whether spaces are acceptable in personalisation fields. Chris thinks they're fine but he's going to run tests tomorrow to double check.

4. Added content on telling recipients who your NHS App message is from to /using-nhs-notify/nhs-app-templates

**Review by jakecosgrove — APPROVED** [review]
